### PR TITLE
Updating copy for adding owners and tooltip

### PR DESCRIPTION
**Pull request by escobarjonatan**
image

### CODE REVIEW

File: src/components/minting/SelectCoCreators.tsx [hunk 1]
```diff
@@ -137,6 +139,30 @@ const FormContent: FunctionComponent<FormContentProps> = ({
         </Button>
       </Stack>
 
+      <Stack mt={ 1.5 } flexDirection="row">
+        <Typography variant="subtitle2">
+          For every additional collaborator who will receive royalties, the fee
+          to complete the minting process will increase by 1.5 ADA.
```
Comment (dmkirshon) [inline src/components/minting/SelectCoCreators.tsx:145]: Saw a little discussion back and forth on the CU task and Slack but wanted to get clarity on this. This 1.5 ADA amount isn't set right? Do we have a scheme for these values to be updated? Or is that only happening on a task for integrating the Minting backend process?

File: src/components/minting/SelectCoCreators.tsx [hunk 2]
```diff
@@ -137,6 +139,30 @@ const FormContent: FunctionComponent<FormContentProps> = ({
...
+        </Typography>
+        <Tooltip
+          title={
+            "This cost is increased with each additional collaborator because " +
+            "this ADA needs to travel with each portion of stream tokens in order " +
+            "to complete the split, which is an added extra cost for each transaction."
+          }
+        >
+          <IconButton sx={ { padding: 0 } }>
+            <HelpIcon
```
Comment (dmkirshon) [inline src/components/minting/SelectCoCreators.tsx:155]: Just a note, but can be addressed during on the mobile responsive effort, the position of the help icon looks out of place when below the md (900px) sizing.

**Review by dmkirshon — APPROVED** [review]
LGTM, just two comments for clarity

File: src/components/minting/SelectCoCreators.tsx (hunk 2 above)
Comment (escobarjonatan) [reply src/components/minting/SelectCoCreators.tsx:155]: uhm, i didnt think it looked that bad, but added 8 px spacing 🫡

File: src/components/minting/SelectCoCreators.tsx (hunk 1 above)
Comment (escobarjonatan) [reply src/components/minting/SelectCoCreators.tsx:145]: Correct, it's not set. They are still trying to figure it out and how the collaborators affect the pricing from what i recall. Either way this should be small enough to implement at a later stage once we know the math part

File: src/components/minting/SelectCoCreators.tsx (hunk 2 above)
Comment (dmkirshon) [reply src/components/minting/SelectCoCreators.tsx:155]: Just looked cramped on my screen 😅 looks good!

**Pull request merged**